data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg

```diff
@@ -130,22 +130,22 @@
 
         [message]
             speaker=Hamel
-            message= _ "Oh, there are orcs enough down here, but ye’ll not likely see them; they hide from us like vermin. You see, lad, in these caves there is no more fearsome a foe than a dwarf. Not only do we move through the caves swiftly, but we know them like the backs of our hands. The orcs did indeed manage to swamp our defenses with their sheer numbers all those long years ago, but not before we surprised and slew their leader — the cursed Khazg Black-Tusk. Since then, although the orcs still foul these caves with their stench, their attacks have been uncoordinated and erratic, wi’ the orcs as likely to attack each other as they are to attack us."
+            message= _ "Oh, there are orcs enough down here, but ye’ll not likely see them; they hide from us like vermin. You see, lad, in these caves there is no more fearsome a foe than a dwarf. Not only do we move through the caves swiftly, but we know them like the backs of our hands. All those years ago, the orcs did indeed manage to swamp our defenses with their sheer numbers, but not before we surprised and slew their leader — that cursed Khazg Black-Tusk. Since then, although the orcs still foul these caves with their stench, their attacks have been uncoordinated and erratic, wi’ the orcs as likely to attack each other as they are to attack us."
         [/message]
 
         [message]
             speaker=Hamel
-            message= _ "In more recent years, however, I ha’ noticed that there have been fewer orcs and more skeletons around. The orcs seem to fear them and shy away from them whenever possible, but their troll allies are another matter. Yon trolls actually seem to take great pleasure out of crushing the skeletons to the dust and sucking any marrow that remains in their bones..."
+            message= _ "In more recent years, however, I ha’ noticed that there have been fewer orcs and more skeletons around. The orcs seem to fear them and shy away from them whenever possible, but their troll allies are another matter. Yon trolls actually seem to take great pleasure out of crushing the skeletons to the dust and sucking what marrow remains in their bones."
         [/message]
 
         [message]
             speaker=Tallin
-            message= _ "What is the story with those skeletons anyway? They look like they were dwarves when they lived, but now they just seem to be mindless killers, attacking anyone and anything."
+            message= _ "What is the story with those skeletons anyway? They look like they were dwarves when they lived, but now they just seem to be mindless killers."
```

Comment:
I like this one.

Reply:
Thanks!

I find in my own writing and others that people write filler when they are thinking about what to say next. A good editing pass can prune those sentences.